### PR TITLE
[IA-665] New FAQ added

### PR DESCRIPTION
**Pull request by mariateresaventura**
New faq added for Certificazione di esenzione.

### CODE REVIEW

File: contextualhelp/data.json [hunk 1]
```diff
@@ -209,6 +213,10 @@
                   "title": "Dove posso salvare il certificato sull'app IO?",
                   "body": "Premendo il pulsante 'Salva', puoi salvare il QR code del certificato nella galleria immagini del tuo dispositivo per poterlo usare anche se non sei connesso ad internet."
                },
+               {
+                  "title": "Posso ricevere su IO la Certificazione di esenzione dalla vaccinazione anti-COVID-19?",
+                  "body": "Sì, puoi ricevere e visualizzare il tuo Certificato di esenzione direttamente in app.\n Questa certificazione è valida solo in Italia fino alla data indicata sulla certificazione e può essere utilizzata per accedere dove è richiesta una Certificazione Verde COVID-19.\n Per maggiori informazioni sulla Certificazione di esenzione dalla vaccinazione anti-COVID-19, consulta le FAQ dedicate [https://www.dgc.gov.it/web/esenzione.html](https://www.dgc.gov.it/web/esenzione.html)."
```
Comment (pietro909) [inline contextualhelp/data.json:218]: I'm getting an error with this URL, is it correct @mariateresaventura ?

Comment (mariateresaventura) [reply contextualhelp/data.json:218]: I check it, the correct url is as follows: https://www.dgc.gov.it/web/esenzione.html

**Comment by pietro909** [comment]
LGTM though I'm not sure who is in charge here for the _go live_ 🤷‍♂️

**Comment by AliceAzzolini** [comment]
Usually we have them approved by @debiff or @Undermaken. The content itself has already been approved by Matteo De Santi.

**Review by pietro909 — APPROVED** [review]
Ok folks, gotta take some ownership here 😨

**Pull request merged**